web/controllers/screeningController/index.ts

```diff
@@ -405,4 +406,125 @@ export async function updateCourse(req: Request, res: Response) {
         return res.status(500).send("internal server error");
     }
 }
+/**
+ * @api {GET} /screening/instructors getInstructors
+ * @apiVersion 1.0.1
+ * @apiDescription
+ *
+ * Retrieves the first 20 courses that match the specified filters.
+ *
+ *
+ * Only screeners with a valid token in the request header can use the API.
+ *
+ * @apiName getInstructors
+ * @apiGroup Screener
+ *
+ * @apiUse Authentication
+ *
+ * @apiExample {curl} Curl
+ * curl -k -i -X GET -H "Token: <AUTHTOKEN>" [host]/api/screening/instructors
+ *
+ * @apiParam (URL Query) {string} screeningStatus get instructors with a certain screeningStatus
+ * @apiParam (URL Query) {string} search fuzzy search inside the instructors name and email, supporting Postgres ILIKE syntax
+ */
+export async function getInstructors(req: Request, res: Response) {
+    try {
+        let { screeningStatus, search } = req.query;
+
+        if ([ScreeningStatus.Accepted, ScreeningStatus.Rejected, ScreeningStatus.Unscreened].indexOf(screeningStatus) === -1)
+            return res.status(400).send("invalid value for parameter 'screeningStatus'");
+
+        if (typeof search !== "string")
+            return res.status(400).send("invalid value for parameter 'search'");
+
+        search = `%${search}%`; // fuzzy search
+
+        let instructors: {}[];
+
+        if (screeningStatus === ScreeningStatus.Accepted) {
+            instructors = await getManager()
+                .createQueryBuilder(Student,"student")
+                .leftJoinAndSelect("student.screening", "screening")
```

Comment:
https://github.com/corona-school/backend/blob/master/common/entity/InstructorScreening.ts should be used for storing instructor screenings. We would like to decouple the normal tutor screening from the instructor screening. The `isInstructor` and `isStudent` booleans were set during registration and will indicate what a user deserves to be, not what he will allowed to be.

Reply:
So if someone will be allowed to be instructor and student/tutor, he will have two screenings, one `InstructorScreening` and a normal `Screening`. This is how it is modeled in the backend right now.

Reply:
Oh cool, thanks for notifying the team working on this so early! :)

Reply:
Sorry about that, I thought, you had see that we have this. #75 was related to that. It was modeled in the backend before you started working on that instructor screening tool. And I thought it was merged before you started working on the instructor screening feature. At the end, it is just a question of how the others modeled that in the backend. Sorry for that miscommunication. It wasn't initially planned to force tutors/students to have an additional screening for being instructor. That was the reason why this was added in such a strange way and probably also wasn't clearly communicated.